api/v1beta2/ibmvpcmachine_types.go

```diff
@@ -126,18 +142,44 @@ type IBMVPCMachineStatus struct {
 	// INSERT ADDITIONAL STATUS FIELD - define observed state of cluster
 	// Important: Run "make" to regenerate code after modifying this file
 
+	// InstanceID defines the IBM Cloud VPC Instance UUID.
+	// +optional
 	InstanceID string `json:"instanceID,omitempty"`
 
 	// Ready is true when the provider resource is ready.
 	// +optional
 	Ready bool `json:"ready"`
 
-	// Addresses contains the GCP instance associated addresses.
+	// Addresses contains the IBM Cloud instance associated addresses.
 	Addresses []corev1.NodeAddress `json:"addresses,omitempty"`
 
-	// InstanceStatus is the status of the GCP instance for this machine.
+	// Conditions deefines current service state of the IBMVPCMachine.
+	// +optional
+	// +k8s:conversion-gen=false
+	Conditions capiv1beta1.Conditions `json:"conditions,omitempty"`
+
+	// FailureReason will be set in the event that there is a terminal problem
+	// reconciling the Machine and will contain a succinct value suitable
+	// for machine interpretation.
+	// +optional
+	// +k8s:conversion-gen=false
+	FailureReason *errors.MachineStatusError `json:"failureReason,omitempty"`
+
+	// FailureMessage will be set in the event that there is a terminal problem
+	// reconciling the Machine and will contain a more verbose string suitable
+	// for logging and human consumption.
+	// +optional
+	// +k8s:conversion-gen=false
+	FailureMessage *string `json:"failureMessage,omitempty"`
+
+	// InstanceStatus is the status of the IBM Cloud instance for this machine.
 	// +optional
 	InstanceStatus string `json:"instanceState,omitempty"`
+
+	// LoadBalancerPoolMembers is the status of IBM Cloud VPC Load Balancer Backend Pools the machine is a member.
+	// +optional
+	// +k8s:conversion-gen=false
+	LoadBalancerPoolMembers []VPCLoadBalancerBackendPoolMember `json:"loadBalancerPoolMembers,omitempty"`
```

Comment:
Is it required here? Is it just to verify that the machine is already registered to LB?

Reply:
Yes, that is the intention, to track and make sure the Machine is in the required pools.